package.json

```diff
@@ -36,29 +36,32 @@
     "walletconnect"
   ],
   "dependencies": {
-    "@ethersproject/sha2": "5.7.0",
-    "@solana/web3.js": "1.63.0",
-    "@walletconnect/qrcode-modal": "1.8.0",
-    "@walletconnect/universal-provider": "^2.1.0",
+    "@ethersproject/sha2": "^5.7.0",
+    "@walletconnect/universal-provider": "^2.1.3",
     "bn.js": "5.2.1",
     "borsh": "0.7.0",
     "bs58": "5.0.0",
+    "buffer": "^6.0.3",
     "valtio": "1.7.0"
   },
   "peerDependencies": {
-    "buffer": "6.0.3"
+    "@solana/web3.js": "1.63.0",
+    "@walletconnect/qrcode-modal": "1.8.0"
```

Comment:
Can this be replaced with standalone v2 modal now?

Reply:
I'll do that in the next PR